src/Module/Register.php

```diff
@@ -86,7 +87,7 @@ protected function content(array $request = []): string
 			}
 		}
 
-		if (!DI::userSession()->getLocalUserId() && (intval(DI::config()->get('config', 'register_policy')) === self::CLOSED)) {
+		if (!DI::userSession()->getLocalUserId() && (Security::getRegisterPolicy() === self::CLOSED)) {
```

Comment:
Save the parentheses!
```suggestion
		if (!DI::userSession()->getLocalUserId() && Security::getRegisterPolicy() === self::CLOSED) {
```